crates/fuel-streams-publisher/src/outputs.rs

```diff
@@ -0,0 +1,135 @@
+use fuel_core_types::fuel_tx::{
```

Comment:
If you run `make start` (run docker compose) you will see the publisher is getting errors because also you are using wrong types, instead of importing from `fuel_stream_core` you are importing directly from `fuel_core`

Please, use a `types.rs` file inside the `outputs/` in the `fuel_streams_core` to avoid the need to import `field::Outputs` here and just import from the core as single source